### PR TITLE
API-2168 - Create test user

### PR DESCRIPTION
**Pull request by tom59**

### CODE REVIEW

File: test/unit/uk/gov/hmrc/testuser/services/TestUserServicesPEC.scala [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    when(underTest.testUserRepository.createUser(Matchers.any[TestUser]())).thenAnswer(sameUser)
+  }
+
+  "createTestIndividual" should {
+
+    "Generate an individual and save it in the database" in new Setup {
+
+      given(underTest.generator.generateTestIndividual).willReturn(testIndividual)
+
+      val result = await(underTest.createTestIndividual)
```
Comment (edinhodzic) [inline test/unit/uk/gov/hmrc/testuser/services/TestUserServicesPEC.scala]: The `createTestIndividual` has a side effect so by Scala conventions it should be called using parenthesis. Same goes for the `createTestOrganisation` invocation.

File: test/it/uk/gov/hmrc/CreateTestUserSpec.scala [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    server = new TestServer(port, app)
+    server.start()
+  }
+
+  def stopServer() = {
+    server.stop()
+  }
+
+  def mongoRepository = {
+    implicit val mongo = new MongoConnector("mongodb://localhost:27017/api-platform-test-user-it").db
```
Comment (edinhodzic) [inline test/it/uk/gov/hmrc/CreateTestUserSpec.scala]: The `new` is probably unnecessary.

File: test/unit/uk/gov/hmrc/testuser/services/TestUserServicesPEC.scala (hunk 1 above)
Comment (tom59) [reply test/unit/uk/gov/hmrc/testuser/services/TestUserServicesPEC.scala]: done

File: test/it/uk/gov/hmrc/CreateTestUserSpec.scala (hunk 2 above)
Comment (tom59) [reply test/it/uk/gov/hmrc/CreateTestUserSpec.scala]: done

**Pull request merged**